### PR TITLE
Fix queries to use date ranges & v2 datasets

### PR DESCRIPTION
**Pull request by alex-deaconu**
fix typo in folder name

### CODE REVIEW

**Comment by alex-deaconu** [comment]
Commits are pretty self explanatory. However, these are the only 3 queries that I found using the Storage Logs data. Let me know if you know of any other queries/tables that I have missed.

@ezequielc @fjvallarino please review. Thanks!

File: projects/rise-core-log/datasets/storage/storageInstallerUsage.bq [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  DATE(TIMESTAMP(time_micros)) as date, 
+  sum(sc_bytes) as bandwidth
+
+FROM TABLE_DATE_RANGE([avid-life-623:RiseStorageLogs_v2.UsageLogs], 
+  TIMESTAMP(DATE_ADD(CURRENT_TIMESTAMP(), -90, 'DAY')),
+  TIMESTAMP(NOW())
+)
+  
+where time_micros > DATE_ADD(CURRENT_TIMESTAMP(), -90, 'DAY')
+and cs_bucket = 'install-versions.risevision.com'
```
Comment (fjvallarino) [inline projects/rise-core-log/datasets/storage/storageInstallerUsage.bq]: @alex-deaconu any reason for not grouping and sorting this one?

Comment (alex-deaconu) [inline projects/rise-core-log/datasets/storage/storageInstallerUsage.bq]: @fjvallarino no, totally missed that, good catch! It was there in the original query. Added back in. Thanks!

**Comment by fjvallarino** [comment]
:+1:

**Pull request merged**